csharp-basic-topics/DetectKeyPressInCSharp/Tests/ConsoleAppTests.cs

```diff
@@ -0,0 +1,50 @@
+using DetectKeyPressInCSharp;
+using Moq;
+using NUnit.Framework;
+
+namespace Tests
+{ 
+    [TestFixture]
+    public class ConsoleAppTests
+    {
+        private Mock<IConsoleService> _mockConsoleService;
+        private ConsoleApp _app;
+
+        [SetUp]
+        public void Setup()
+        {
+            _mockConsoleService = new Mock<IConsoleService>();
+            _app = new ConsoleApp(_mockConsoleService.Object);
+        }
+
+        [Test]
+        public void KeyInputs_ReadKeyUseIsCalled_ShouldPrintCorrectMessages()
```

Comment:
@mohitkumar342 
Feedback from the senior editor is that `Our tests still aren’t following Given_When_Then.`
We can modify this as `WhenConsoleReadkey_ThenPrintCorrectMessage`

Reply:
This is done,